flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
that's disappointing.

Reply:
I haven't attempted any cross compiles, I suppose I should leave it up to the bot though.

Reply:
After checking, the bin that I'm using doesn't support anything other than x86_64. I did only skim the website though, so unsure if arm64 support exists elsewhere.